### PR TITLE
Changing automations in cylance playbooks

### PR DESCRIPTION
**Pull request by evisochek**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/35985

## Description
Need to change all the automations in Cylance playbooks, in the current situation even if Cylance Protect v2 integration is enabled when trying to run the Cylance playbooks it will fail due to error that the automation is disabled.
Need to move Get File Sample By Hash - Cylance Protect playbook to deprecate content

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Cylance_Protect/Playbooks/playbook-Endpoint_Enrichment_-_Cylance_Protect_v2.yml [hunk 1]
```diff
@@ -84,19 +88,23 @@ tasks:
...
+    taskid: 79a6eab0-7d4b-49d2-8348-7abcfbec8bb5
     type: regular
     task:
-      id: 6fc275f9-cd51-4d40-8fe4-52a92bbc6af7
+      id: 79a6eab0-7d4b-49d2-8348-7abcfbec8bb5
       version: -1
       name: Get all Cylance Protect devices
-      description: Gets information about all devices that are available in Cylance Protect.
-      script: Cylance Protect v2|||cylance-protect-get-devices
+      description: Allows a caller to request a page with a list of Console device
```
Comment (David-BMS) [inline Packs/Cylance_Protect/Playbooks/playbook-Endpoint_Enrichment_-_Cylance_Protect_v2.yml:101]: What was the change here? seems like a new description

Comment (evisochek) [reply Packs/Cylance_Protect/Playbooks/playbook-Endpoint_Enrichment_-_Cylance_Protect_v2.yml]: Fiexed.

File: Packs/Cylance_Protect/Playbooks/playbook-Get_File_Sample_By_Hash_-_Cylance_Protect_README.md [hunk 2]
```diff
@@ -7,33 +7,32 @@ This playbook uses the following sub-playbooks, integrations, and scripts.
...
 ## Playbook Inputs
 ---
 
-| **Name** | **Description** | **Default Value** | **Source** | **Required** |
-| --- | --- | --- | --- | --- |
-| SHA256 | The SHA256 hash of the file to download. | SHA256 | File | Optional |
-| ZipPassword | The password for the zip file. | infected | - | Required |
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| SHA256 | The SHA256 hash of the file we want to download | File.SHA256 | Optional |
```
Comment (David-BMS) [inline Packs/Cylance_Protect/Playbooks/playbook-Get_File_Sample_By_Hash_-_Cylance_Protect_README.md:26]: @evisochek looks like a mismatch with the actual playbook

File: Packs/DeprecatedContent/pack_metadata.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "name": "Deprecated Content",
     "description": "Deprecated Cortex XSOAR content pack.",
     "support": "xsoar",
-    "currentVersion": "1.6.7",
```
Comment (David-BMS) [inline Packs/DeprecatedContent/pack_metadata.json:5]: @evisochek Wasnt this suppose to be removed?

Comment (evisochek) [reply Packs/DeprecatedContent/pack_metadata.json]: removed now

**Review by David-BMS — APPROVED** [review]

**Pull request merged**